database/migrator.go

```diff
@@ -0,0 +1,70 @@
+package database
+
+import (
+	"database/sql"
+	"fmt"
+	"os"
+
+	"github.com/golang-migrate/migrate/v4"
+	"github.com/golang-migrate/migrate/v4/database"
+	migmysql "github.com/golang-migrate/migrate/v4/database/mysql"
+	migsqlite3 "github.com/golang-migrate/migrate/v4/database/sqlite3"
+	_ "github.com/golang-migrate/migrate/v4/source/pkger"
+	"github.com/markbates/pkger"
+
+	"github.com/moov-io/base/log"
+)
+
+func RunMigrations(log log.Logger, db *sql.DB, config DatabaseConfig) error {
+	if _, err := os.Stat(config.migrationsDir); os.IsNotExist(err) {
+		return fmt.Errorf("migrations directory=\"%s\" does not exist", config.migrationsDir)
+	}
+
+	log.Info().Log("Running Migrations")
+	_ = pkger.Include(config.migrationsDir)
+
+	driver, err := GetDriver(db, config)
+	if err != nil {
+		return err
+	}
+
+	m, err := migrate.NewWithDatabaseInstance(
+		"pkger:///migrations/",
```

Comment:
This one should use `config.migrationsDir` (most probably `path.Base(config.migrationsDir)`)

From migrate package:

```go
pkger.Include("/module/path/to/migrations")
m, err := migrate.New("pkger:///module/path/to/migrations", ...)
```